code/game/machinery/computer/security_alert.dm

```diff
@@ -0,0 +1,109 @@
+var/list/security_alerts_computers = list()
+	
+/****************
+Security Alerts Computer
+This computer is supposed to be the comprehensive list
+of automated alarms that security should be made aware of.
+This includes SPS alarms, lockbox cracking alarms, camera alarms, 
+door alarms, crate cracking alarms, turret engagement warnings,
+encoded proximity alarms, you name it.
+Alerts are listed as TYPE - TIME - AREA + CO,OR,DINATES.
+TODO: literally every alarm but SPS alarms.
+***************/	
+	
+/obj/machinery/computer/security_alerts //copied mostly from the bhangometer, see it for comments
+	name = "Security Alerts Computer"
+	desc = "Lists security alerts from various sensors."
+	circuit = "/obj/item/weapon/circuitboard/security_alerts"
+	icon_state = "secalert"
+	var/list/saved_security_alerts = list()
+
+	light_color = LIGHT_COLOR_RED
+
+/obj/machinery/computer/security_alerts/New()
+	..()
+	security_alerts_computers += src
+
+/obj/machinery/computer/security_alerts/Destroy()
+	security_alerts_computers -= src
+	..()
+
+/obj/machinery/computer/security_alerts/process()
+	return PROCESS_KILL
+
+/obj/machinery/computer/security_alerts/attack_paw(mob/user as mob)
+	return src.attack_hand(user)
+
+/obj/machinery/computer/security_alerts/attack_ai(mob/user as mob)
+	src.add_hiddenprint(user)
+	return src.attack_hand(user)
+
+/obj/machinery/computer/security_alerts/attack_hand(mob/user as mob)
+	add_fingerprint(user)
+	ui_interact(user)
+
+/obj/machinery/computer/security_alerts/update_icon()
```

Comment:
this whole proc is unnecessary since it inherits this behavior from computer

Reply:
not anymore